### PR TITLE
(#149) Fixed README doc on Assertion uasage

### PR DESCRIPTION
**Pull request by fanifieiev**
This PR is for the [issue](https://github.com/llorllale/cactoos-matchers/issues/149)

### CODE REVIEW

**Comment by 0crat** [comment]
@fanifieiev[/z](https://www.0crat.com/u/fanifieiev) this pull request is too small, just 2 lines changed (less than 10), there will be no formal code review, see [§53](http://www.zerocracy.com/policy.html#53) and [§28](http://www.zerocracy.com/policy.html#28); in the future, try to make sure your pull requests are not too small; @paulodamaso[/z](https://www.0crat.com/u/paulodamaso) please review this and merge or reject

**Comment by paulodamaso** [comment]
@fanifieiev Please check if we don't have any other invalid example in README.md

**Comment by fanifieiev** [comment]
@paulodamaso I have fixed some more usages of `Assertion` in the README

File: README.md [hunk 1]
```diff
@@ -79,22 +79,25 @@ public void csvLineHasCorrectFormat() throws Exception {
...
+      ),
+      () -> new TextOf("")
     ).value(),
     new MatchesRegex(fields)
   ).affirm();
 }
 
 @Test
-public void textIsBlank() {
+public void textIsBlank() throws IOException {
```
Comment (paulodamaso) [inline README.md]: @fanifieiev  This code won't work: `TextOf.asString` throws an `IOException` which is not being handled in this example. Also, you can remove the `<>` from `Assertion`

Comment (fanifieiev) [reply README.md]: @paulodamaso Why removing `<>` from `Assertion`? The `Assertion` is a generic type.

**Comment by fanifieiev** [comment]
@paulodamaso Please review again.

**Comment by fanifieiev** [comment]
@paulodamaso Please review

File: README.md (hunk 1 above)
Comment (paulodamaso) [reply README.md]: @fanifieiev Right

**Comment by paulodamaso** [comment]
@rultor merge

**Comment by paulodamaso** [comment]
@fanifieiev Looks like travis-ci is stuck and I can't merge the PR.

**Comment by llorllale** [comment]
@paulodamaso I "unstuck" Travis, you should now merge

**Comment by fanifieiev** [comment]
@paulodamaso Please merge

**Comment by paulodamaso** [comment]
@rultor merge

**Comment by 0crat** [comment]
Job `gh:llorllale/cactoos-matchers#150` is not assigned, can't get performer

**Comment by 0crat** [comment]
This job is not in scope

**Comment by paulodamaso** [comment]
@rultor release tag is ` 0.19`

**Comment by paulodamaso** [comment]
@rultor release tag is `0.19`

**Pull request merged**